linera-service/tests/linera_net_tests.rs

```diff
@@ -3097,6 +3100,11 @@ async fn test_end_to_end_assign_greatgrandchild_chain(config: impl LineraNetConf
     client2.process_inbox(chain2).await?;
     assert!(client2.local_balance(account2).await? > Amount::ZERO);
 
+    // Verify that a third party can also follow the chain.
+    client3.follow_chain(chain2).await?;
```

Comment:
Intuitively, _follow chain_ would start syncing immediately.

Reply:
Good point, let's discuss with the team. (Would be a separate PR, though.)